lms/djangoapps/commerce/http.py

```diff
@@ -13,9 +13,13 @@ def __init__(self, message, status=HTTP_200_OK):
         super(DetailResponse, self).__init__(object=data, status=status)
 
 
-class ApiErrorResponse(DetailResponse):
-    """ Response returned when calls to the E-Commerce API fail or the returned data is invalid. """
+class InternalRequestErrorResponse(DetailResponse):
+    """ Response returned when an internal service request fails. """
 
-    def __init__(self):
-        message = 'Call to E-Commerce API failed. Order creation failed.'
-        super(ApiErrorResponse, self).__init__(message=message, status=HTTP_503_SERVICE_UNAVAILABLE)
+    def __init__(self, internal_message, internal_status):
+        message = (
+            'Call to E-Commerce API failed due to internal service request. Internal Request Status Code: '
```

Comment:
The due to part doesn't seem right. All of the API calls are internal service requests. `Call to E-Commerce API failed.` should be sufficient